File: JS-Basics/Conditional Statements/Exercise/shopping.js
function shopping(input) {
    let budget = Number(input[0]);
    let cardsAmount = Number(input[1]);
    let processorsAmount = Number(input[2]);
    let memoryAmount = Number(input[3]);

    let graphicCard = 250 * cardsAmount;
    let processor = 0.35 * graphicCard;
    let memory = 0.1 * graphicCard;

    let remainingMoney = 0;

    let totalPrice = graphicCard + processor * processorsAmount + memory * memoryAmount;

    if (cardsAmount > processorsAmount) {
        totalPrice *= 0.85;
    }

    if (budget > totalPrice || budget == totalPrice) {
        remainingMoney = budget - totalPrice;
        console.log(`You have ${remainingMoney.toFixed(2)} leva left!`);
    } else {
        remainingMoney = totalPrice - budget;
        console.log(`Not enough money! You need ${remainingMoney.toFixed(2)} leva more!`);
    }
}

shopping(["920.45","3","1","1"]);

File: Python/SoftUni PB/Exercises/Python Basics/Conditional Statements - Exercise/Shopping.py
budget = float(input())
graphic_card_amount = int(input())
processor_amount = int(input())
memory_amount = int(input())

graphic_card = 250 * graphic_card_amount
processor = 0.35 * graphic_card * processor_amount
memory = 0.1 * graphic_card * memory_amount

total_price = graphic_card + processor + memory

if graphic_card_amount > processor_amount:
    total_price -= 0.15 * total_price
else:
    total_price += 0

remaining_money = abs(budget - total_price)

if budget >= total_price:
    print(f"You have {remaining_money:.2f} leva left!")
else:
    print(f"Not enough money! You need {remaining_money:.2f} leva more!")
